extensions/spotify-beta/.eslintrc.json

```
@@ -0,0 +1,10 @@
+{
+  "root": true,
+  "env": {
+    "es2020": true,
+    "node": true
+  },
+  "parser": "@typescript-eslint/parser",
+  "plugins": ["@typescript-eslint"],
+  "extends": ["eslint:recommended", "plugin:@typescript-eslint/recommended", "prettier"]
+}
```

Let's use `@raycast/eslint-config` here:

```suggestion
{
  "root": true,
  "extends": ["@raycast"]
}
```